src/directives/sortable/sortable.ts

```diff
@@ -162,7 +163,7 @@ export class MSortable extends MElementDomPlugin<MSortableOptions> {
                     action: !grouping ? MSortableAction.Move : MSortableAction.MoveGroup,
                     dragData: this.options.items[itemCounter++],
                     grouping,
-                    canDrag: this.options.canSort
+                    canDrag: childDragValue && childDragValue.value === 'false' ? false : this.options.canSort
```

Comment:
Je pense que ça serait plus prudent de faire quelque chose comme : 
canDrag: childDragValue ? childDragValue.value === 'true' : this.options.canSort
Ça éviterait un bug si jamais childDragValue.value est true et this.options.canSort est false.

Reply:
Mais si le childDragValue.value est a false ça fonctionne pas non plus de ta façon aussi?

Reply:
Moi j'avais proposé qu'on se fie simplement au tag "draggable" de l'élément pour initialiser la valeur.

Donc si l'objet à draggable = false on ionitialise pas le draggable
Si draggable = true ou undefined on l'initialise

Reply:
J'ai essayer cette option, mais puisque la valeur est analyser avant son attribution  (this.element.draggable = true;) dans le fichier draggable.ts et bien par défaut la valeur est ta false si elle n'est pas défini dans l'élément Child. J'ai utilisé un autre attribut aussi puisque une fois draguer ton code attribut la valeur a false donc on perdait la valeur initial. Je peux descendre en discuter avec toi si tu veux

Reply:
Dans ce que je propose, si childDragValue.value est false ça va retourner false comme c'est le cas avec ton code déjà.

Reply:
j'ai modifier le code, voir le nouveau code